sonic_master_apply_patch_bullseye.sh

```diff
@@ -29,6 +29,7 @@ WGET_PATH="https://raw.githubusercontent.com/Marvell-switching/sonic-scripts/mas
 # Patches
```

Comment:
"master_01" tag can be changed to "master" now. Otherwise changes will not be picked up for compilation.

Reply:
already done